docs/app.tsx

```diff
@@ -142,31 +170,62 @@ async function graph({
       // benchmark and node-version
       benchmark: [
         BENCHMARKS_ORDER[b.benchmark],
-        NODE_VERSIONS.indexOf(getNodeMajorVersionNumber(b.nodeVersion)),
-        b.nodeVersion,
+        NODE_VERSIONS.indexOf(getNodeMajorVersionNumber(b.runtimeVersion)),
+        b.runtimeVersion,
         b.benchmark,
       ].join('-'),
     }));
 
-  const nodeJsVersionCount = new Set(values.map(v => v.nodeVersion)).size;
+  const valuesBun = benchmarkResultsBun
```

Comment:
I see what you did here :grin:  - you basically copied all the fetching, data-preparation and filter stuff so that there are lists of node and bun benchmarks.

Maybe using a single-array with `Bun | Nodejs` results would result in less code overall.

But your approach works too ofc. Hats off to you for figuring out how this code works and extending it!